Packs/Automox/Integrations/Automox/README.md

```diff
@@ -0,0 +1,5 @@
+This README contains the full documentation for your integration.
```

Comment:
Does this need generated?

Reply:
I'm going to generate this and any other documentation in another story.